config/firebase.js

```diff
@@ -1,23 +0,0 @@
-import firebase from 'firebase/app';
```

Comment:
Add this file back and remove from gitignore. I dont think there is any sensitive data

Reply:
okay ill add that